cache/mock/cache.go

```diff
@@ -1,14 +1,13 @@
 package heptane
 
 import (
-	"bytes"
 	"fmt"
 
-	"github.com/heptanes/heptane"
+	c "github.com/heptanes/heptane/cache"
```

Comment:
should a package alias be more descriptive?

Reply:
which one do you suggest?